chainerx_cc/chainerx/cuda/cuda_device/comparison.cu

```diff
@@ -135,6 +135,54 @@ public:
 
 CHAINERX_REGISTER_OP_CUDA(LogicalNotOp, CudaLogicalNotOp);
 
+template <typename T>
+struct LogicAndImpl {
```

Comment:
A minor but could you align naming convention to `LogicalNotImpl`, i.e.
```suggestion
struct LogicalAndImpl {
```

Reply:
Oh yes. Missed that.